src/components/listing-page/Details/index.jsx

```diff
@@ -0,0 +1,54 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import {
+  Container,
+  DetailsItem,
+  Key,
+  Title,
+  Value,
+  Wrapper,
+} from './details.styles';
+
+/**
+ * Listing page details section.
+ *
+ * @param {Object} props      The component props.
+ * @param {Object} props.data The component data.
+ *
+ * @return {JSX.Element}
+ */
+function Details({ data }) {
+  const { details } = data;
```

Comment:
I see you're passing the data.details directly, so you can get the details only  
```suggestion
function Details({ details }) {
```